Automattic/jetpack
# Boost: Add jetpack compatibility tests

#### Changes proposed in this Pull Request:

* Add tests for Jetpack compatibility
* List according to #22019:
    * Core Test Id 9
    * Core Test Id 10
* To retrieve the core Jetpack module status, this PR also adds a function `isModuleActive` to e2e-commons - prerequisites.js

#### Jetpack product discussion


Slack : archives/C0232B8CUHG/p1639639809027900

#### Does this pull request change what data or activity we track or use?


No

#### Testing instructions:






* Run tests as described here: #21819
* Verify that all the tests pass including the ones added in this PR!
